onnxscript/rewriter/ort_fusions/_test_utils.py

```diff
@@ -33,7 +33,7 @@ def ort_run(model_name: str, model, inputs):
     return session.run(None, inputs)
 
 
-def assert_allclose(outputs, expected_outputs, rtol=1e-2, atol=1e-2):
+def assert_allclose(outputs, expected_outputs, rtol=1e-4, atol=1e-4):
     for i, (baseline_output, optimized_output) in enumerate(zip(expected_outputs, outputs)):
         try:
             np.testing.assert_equal(baseline_output.shape, optimized_output.shape)
```

Comment:
```suggestion
```

Possible to set `strict=True` instead to compare shape and dtype https://numpy.org/doc/2.2/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose

Reply:
Hmmm ... that is new in numpy version 2.0. So, I am going to undo this change.

Reply:
I see ok

Reply:
But this is test so I guess that's ok?